Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -195,4 +195,80 @@ final class GINIMultipageReviewControllerTests: XCTestCase {
+                       "delete button should be disabled when tooltip is shown")
+        
+    }
+    
+    func testDeleteButtonEnabledWhenToolTipIsNotShown() {
+        ToolTipView.shouldShowReorderPagesButtonToolTip = false
+        
+        multipageReviewViewController = MultipageReviewViewController(imageDocuments: imageDocuments,
+                                                                  giniConfiguration: giniConfiguration)
+        _ = multipageReviewViewController.view
+        multipageReviewViewController.viewDidAppear(false)
+        
+        XCTAssertTrue(multipageReviewViewController.deleteButton.isEnabled,
+                       "delete button should be disabled when tooltip is shown")
+        
+    }
+    
+    func testRotateButtonDisabledWhenToolTipIsShown() {
+        ToolTipView.shouldShowReorderPagesButtonToolTip = true
+
+        multipageReviewViewController = MultipageReviewViewController(imageDocuments: imageDocuments,
+                                                                  giniConfiguration: giniConfiguration)
+        _ = multipageReviewViewController.view
+        multipageReviewViewController.viewDidAppear(false)
+
+        XCTAssertFalse(multipageReviewViewController.rotateButton.isEnabled,
+                       "rotate button should be disabled when tooltip is shown")
+        
+    }
+    
+    func testRotateButtonEnabledWhenToolTipIsNotShown() {
+        ToolTipView.shouldShowReorderPagesButtonToolTip = false
+        
+        multipageReviewViewController = MultipageReviewViewController(imageDocuments: imageDocuments,
+                                                                  giniConfiguration: giniConfiguration)
+        _ = multipageReviewViewController.view
+        multipageReviewViewController.viewDidAppear(false)
+        
+        XCTAssertTrue(multipageReviewViewController.rotateButton.isEnabled,
+                       "rotate button should be disabled when tooltip is shown")
+        
+    }
+    
+    func testToolTipShouldAppearTheFirstTime() {
+        ToolTipView.shouldShowReorderPagesButtonToolTip = true
+        
+        multipageReviewViewController = MultipageReviewViewController(imageDocuments: imageDocuments,
+                                                                  giniConfiguration: giniConfiguration)
+        _ = multipageReviewViewController.view
+        
+        XCTAssertNotNil(multipageReviewViewController.toolTipView,
+                     "rotate button should be disabled when tooltip is shown")
+        
+    }
+    
+    func testToolTipShouldNotAppearWhenItWasShownBefore() {
+        ToolTipView.shouldShowReorderPagesButtonToolTip = false
+        
+        multipageReviewViewController = MultipageReviewViewController(imageDocuments: imageDocuments,
+                                                                  giniConfiguration: giniConfiguration)
+        _ = multipageReviewViewController.view
```

Comment:
These 3 lines of code are the same in at least 4-5 tests. Can you put them in a method? `simulateViewControllerAppearing` or something?